packages/document/main-doc/docs/en/blog/index.md

```diff
@@ -4,5 +4,70 @@ sidebar_position: 1
 
 # Blog
 
-You can find the latest news about Modern.js here.
+Find the latest update of modern.js and our thought processes here. Occasionally we explain the infrastructure behind it.
 
+---
+
+## Releasing MOdern.js v2: support Rspack
+
+> Published on 16.03.2023
+
+Hiyaaaaa~~ we are excited to announce the release of Modern.js v2!
+
+Modern.js is a suite of infrastructure tools we built for web development inside Bytedance (we call ourself Web-Infra-Dev internally...). Since we open sourced this project a little more than a year ago, there were dozens of contributors helped us on development; we aggregated more than 2,000 pull requests and modern.js started to support build tool like Rspack, nested routes, SSR(Streaming Server Render).
```

Comment:
SSR is usually an abbreviation for "Server-Side Rendering", so here we should use "Streaming SSR".

```suggestion
Modern.js is a suite of infrastructure tools we built for web development inside Bytedance (we call ourself Web-Infra-Dev internally...). Since we open sourced this project a little more than a year ago, there were dozens of contributors helped us on development; we aggregated more than 2,000 pull requests and modern.js started to support build tool like Rspack, nested routes, Streaming SSR.
```

Reply:
Since this is the case, we should start with `SSR(Server-side Rendering)` at the beginning of this documentation at line 27. But agree.